components/List.tsx

```diff
@@ -1,23 +1,37 @@
-import * as React from "react"
+import * as React from 'react'
```

Comment:
I think we configured TypeScript to import as ES6 syntax. So `import React from 'react'` should work.